docs/ops.html

```diff
@@ -2032,17 +2032,25 @@ <h5 class="anchor-heading"><a id="kafka_streams_store_monitoring" class="anchor-
  </table>
 
   <h5 class="anchor-heading"><a id="kafka_streams_rocksdb_monitoring" class="anchor-link"></a><a href="#kafka_streams_rocksdb_monitoring">RocksDB Metrics</a></h5>
-  All of the following metrics have a recording level of <code>debug</code>.
-  The metrics are collected every minute from the RocksDB state stores.
-  If a state store consists of multiple RocksDB instances as it is the case for aggregations over time and session windows,
-  each metric reports an aggregation over the RocksDB instances of the state store.
+  RocksDB Metrics are grouped into statistics-based metrics and properties-based metrics.
+  The former are recorded from statistics that a RocksDB state store collects whereas the latter are recorded from
+  properties that RocksDB exposes.
```

Comment:
I'm still a little confused about the difference between the two after reading this. Maybe an example of something that is exposed by RocksDB but not collected by RocksDB would help

Reply:
I elaborated on the two types of metrics. Let me know if it is better now.

Reply:
Thanks for the explanation, I also didn't understand the difference at first

Reply:
Ahh I see now, thanks for the additional info! LGTM